Makefile

```diff
@@ -153,6 +186,16 @@ test:
 	fi
 	bash scripts/check-bins-for-jemalloc.sh
 
+# Unlike test, this target will trace tests and output logs when fail
+# test is detected.  It's not clear who uses this and for what. It
+# does not seem to be used by CI. If you know please document it.
+trace_test:
```

Comment:
Can we just remove it and re-add it if someone complains? I remember it was used in circleCI but haven't seen it used otherwise.

Reply:
I have done so.